openwisp_monitoring/device/base/models.py

```diff
@@ -454,6 +455,25 @@ def handle_disabled_organization(cls, organization_id):
             status='unknown'
         )
 
+    @classmethod
+    def _get_critical_metric_keys(cls):
+        from openwisp_monitoring.device.settings import get_critical_device_metrics
```

Comment:
Please move this to the top imports of the file, there's no good reason to import this here.